qiskit/test/mock/fake_openpulse_2q.py

```diff
@@ -93,22 +93,22 @@ def __init__(self):
                                             samples=7*[0.j, 0.1j, 1j, 0.5 + 0j])],
             cmd_def=[Command(name='u1', qubits=[0],
                              sequence=[PulseQobjInstruction(name='fc', ch='d0',
-                                                            t0=0, phase='-P1*np.pi')]),
+                                                            t0=0, phase='-P1')]),
```

Comment:
this is usually defined as `P0`. same for others.

Reply:
fixed in https://github.com/Qiskit/qiskit-terra/pull/3785/commits/31143c4b2cc47e17047ea155c192cde98190d7f3